### PR TITLE
feat: include log for each database name in export workflow

### PR DESCRIPTION
**Pull request by adeusmondo**
This PR is to add log, because sometimes i need to show how many time each database takes to be exported

### CODE REVIEW

**Comment by comment-bot-dev** [comment]
@adeusmondo
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@adeusmondo
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

File: modules/backup/variables.tf [hunk 1]
```diff
@@ -102,3 +102,9 @@ variable "unique_suffix" {
   type        = string
   default     = ""
 }
+
+variable "enable_database_name_log_export" {
```
Comment (g-awmalik) [inline modules/backup/variables.tf:106]: Does `log_db_name_to_export` sound better?

File: modules/backup/main.tf [hunk 2]
```diff
@@ -110,6 +110,7 @@ resource "google_workflows_workflow" "sql_export" {
     gcsBucket           = var.export_uri
     dbType              = split("_", data.google_sql_database_instance.backup_instance.database_version)[0]
     compressExport      = var.compress_export
+    databaseNameLog     = var.enable_database_name_log_export
```
Comment (g-awmalik) [inline modules/backup/main.tf:113]: Hows `logDbName` for this var?

**Review by g-awmalik — COMMENTED** [review]
Thanks for the PR. A few minor nits.

**Comment by adeusmondo** [comment]
> Thanks for the PR. A few minor nits.

Oh, sorry for the delay and thanks for the code review

File: modules/backup/main.tf (hunk 2 above)
Comment (adeusmondo) [reply modules/backup/main.tf:113]: Yes! sounds better and I believe it is clearer to understand

File: modules/backup/variables.tf (hunk 1 above)
Comment (adeusmondo) [reply modules/backup/variables.tf:106]: Yes!

**Comment by comment-bot-dev** [comment]
@adeusmondo
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=autogen' '--exclude=*.tfvars' '--exclude=*metadata.yaml' /workspace/modules/backup/README.md /tmp/tmp.M4GZzhZShv/generate_docs/workspace/modules/backup/README.md
59d58
< | enable\_database\_name\_log\_export | Whether or not to log database name in the export workflow | `bool` | `false` | no |
64a64
> | log\_db\_name\_to\_export | Whether or not to log database name in the export workflow | `bool` | `false` | no |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ```

**Comment by comment-bot-dev** [comment]
@adeusmondo
Thanks for the PR! 🚀
Unfortunately it looks like some of our CI checks failed. See the [Contributing Guide](../blob/master/CONTRIBUTING.md) for details.
- ⚠️check_documentation
 The documentation needs to be regenerated. Please run `make generate_docs`. 
 ```bash 
Checking for documentation generation
diff -r '--exclude=.terraform' '--exclude=.kitchen' '--exclude=autogen' '--exclude=*.tfvars' '--exclude=*metadata.yaml' /workspace/modules/backup/README.md /tmp/tmp.eETQNCnP0P/generate_docs/workspace/modules/backup/README.md
59d58
< | enable\_database\_name\_log\_export | Whether or not to log database name in the export workflow | `bool` | `false` | no |
64a64
> | log\_db\_name\_to\_export | Whether or not to log database name in the export workflow | `bool` | `false` | no |
Error: Documentation generation has not been run, please run the
'make docker_generate_docs' command and commit the above changes.
 ```

**Comment by comment-bot-dev** [comment]
@adeusmondo
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@adeusmondo
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Comment by comment-bot-dev** [comment]
@adeusmondo
Thanks for the PR! 🚀<br/>✅ Lint checks have passed.

**Review by g-awmalik — APPROVED** [review]

**Pull request merged**